app/src/main/java/cs2063/groceryshopper/AddActivity.kt

```diff
@@ -0,0 +1,226 @@
+package cs2063.groceryshopper
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import android.view.View
+import android.widget.AdapterView
+import android.widget.AdapterView.OnItemLongClickListener
+import android.widget.ArrayAdapter
+import android.widget.Button
+import android.widget.EditText
+import android.widget.LinearLayout.LayoutParams
+import android.widget.ListView
+import android.widget.RelativeLayout
+import android.widget.Spinner
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import cs2063.groceryshopper.util.DBHelper
+import cs2063.groceryshopper.util.ListOfEditorsGenerator
+import cs2063.groceryshopper.util.ListOfItemsGenerator
+import java.util.ArrayList
+import java.util.Locale
+import java.util.concurrent.Executors
+import java.util.regex.Pattern
+
+// ToDo: Clean this file up
+
+class AddActivity : AppCompatActivity() {
+
+    private lateinit var dbGlobal: DBHelper
+    private lateinit var itemsListGlobal: ArrayList<List<String>>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.add_view)
+
+        // Setup Back Button
+        this.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+
+//        tripPriceTotal = intent.extras?.getDouble("total")!!
+
+        // ToDo: Fix title bar (not sure what to put there)
+        // Update Titles
+//        val title = this.findViewById<TextView>(R.id.dateTitle)
+//        val actionbarString = intent.extras?.getString("storeName") + "  |  " + intent.extras?.getString("date")
+//        val titleString = "Total Spent: " + "%,.2f".format(Locale.ENGLISH, tripPriceTotal) + "$"
+//        this.supportActionBar?.title = actionbarString
+//        title.text = titleString
+
+        // Setup Dropdown
+//        setupSpinner()
```

Comment:
I'd say remove these if we aren't using them.

If you want to change the title on the actionbar at the top just set:
val actionbarString = "String-Of-Title-Here"
this.supportActionBar?.title = actionbarString